pkg/rtc/transport.go

```diff
@@ -131,6 +132,30 @@ func (e event) String() string {
 
 // -------------------------------------------------------
 
+type wrappedICECandidatePairLogger struct {
+	pair *webrtc.ICECandidatePair
+}
+
+func (w wrappedICECandidatePairLogger) MarshalLogObject(e zapcore.ObjectEncoder) error {
+	if w.pair == nil {
+		return nil
+	}
+
+	if w.pair.Local != nil {
+		e.AddString("localProtocol", w.pair.Local.Protocol.String())
```

Comment:
wish they had a `CloseNamespace`. They have `OpenNamespace`. I can do something like `OpenNamespace("local")` and the following fields will get grouped. But, there is no way to close it. So, if I add another `OpenNamespace("remote")` below, it get nested further.